README.md

```diff
@@ -1 +1,172 @@
-# Portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>PERSONAL PORTFOLIO</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [PERSONAL PORTFOLIO] <a name="about-project"></a>
+
+> A "portfolio" HTML-5 & CSS3 repository built during the portal learning module.
+
+It is a show of my understanding of Gitflow, use of Linters, HTML5 semantics, CSS3 flexbox
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML-5
+- CSS3
+- Linters
+- Figma
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.testim.io/blog/what-is-a-linter-heres-a-definition-and-quick-start-guide/">Linters</a></li>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Key features of the application.
+
+- **[Navigation bar]**
+- **[Header section - Introduction]**
+- **[Work Portfolio section - Project description]**
+- **[About Link]**
+- **[Contact Link]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- A code editor of your choice (like vs code or atom and so on)
+-version control system (git is preferred)
+
+### Setup
+
+- Create a new folder to store the files in.
+
+### Install
+
+Install this project with:
+
+- Open your command prompt or terminal and run.
+-  git clone -b feature https://github.com/siddghosh108/Portfolio.git
+cd -Portfolio-
+
+### Usage
+
+- this is my portflio and will use to insert my upcoming projects.
+
+### Run tests
+
+To run tests, run the following command:
+
+- To run tests run :
+- npx hint .
+- npx stylelint "\*_/_.{css,scss}"
+
+### Deployment
+
+- Project is deployed using [GitHub Pages](https://pages.github.com/)
+- [🚀 Live Demo Coming Soon!]
+
```

Comment:
- [ ] Please as suggest by the previous reviewer, kindly remove the link since this project has not been deployed yet. 😄